assets/queries/terraform/alicloud/ram_password_security_settings_password_without_reuse_prevention/metadata.json

```
@@ -0,0 +1,11 @@
+{
+    "id": "a8128dd2-89b0-464b-98e9-5d629041dfe0",
+    "queryName": "RAM Password Security Settings Password without Reuse Prevention",
```

```suggestion
    "queryName": "Ram Account Password Policy without Reuse Prevention",
```
Please, also update the query folder name to `ram_account_password_policy_without_reuse_prevention` and update the positive_expected_value.json